amlogic.cmake

```diff
@@ -165,3 +165,5 @@ endif()
 if(BUILD_ENABLE_APP_CONTROL_AUDIOPORT_INIT)
    add_definitions (-DAPP_CONTROL_AUDIOPORT_INIT)
 endif()
+
+add_definitions (-DDCM_REMOVAL)
```

Comment:
Please check whether it is DCM_REMOVAL or DCM_REMOVAL_MM

Reply:
As discussed, modified the flag to DCM_TASK_REMOVAL. 
Validated the functionality. Working fine.